services/nextjs-app/next-env.d.ts

```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
+
+// NOTE: This file should not be edited
+// see https://nextjs.org/docs/basic-features/typescript for more information.
```

Comment:
I see a lot of the env and config file are like this where it's all commented out. I will trust that that it all generated code and we leave it as is!

Reply:
Yup, it was generated